packages/amplify-opensearch-simulator/package.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
is this a new package? is there a reason this is not written in typescript?

Reply:
Yes it is a new package but a bit similar to the existing dynamodb simulator package. I wanted to keep the logic to start/stop the emulator in available ports, same as we have in other place in JS, since I was running into errors re-writing it in TS.